packages/marko-web-theme-monorail/components/blocks/section-card-list.marko

```diff
@@ -14,6 +14,10 @@ $ const queryParams = {
 $ const blockName = "section-card-list";
 $ const buttonClass = input.buttonClass || "btn btn-secondary";
 $ const linkHeader = input.linkHeader || false;
+$ const imageProps = input.imageProps;
+$ const imageFluid = typeof imageProps.fluid === 'boolean' ? imageProps.fluid : true;
+$ const imageWidth = imageProps.width ? imageProps.width : 340;
+$ const imageAspectRatio = imageProps.ar ? imageProps.ar : "3:2"
```

Comment:
Use `getAsObject` (and `defaultValue` for consistency) helpers here. If `imageProps` is not passed, this will throw a hard error trying to access `fluid`, `width`, or `ar` properties.